src/schema/city/index.ts

```diff
@@ -50,10 +50,20 @@ const CityType = new GraphQLObjectType<any, ResolverContext>({
           defaultValue: "CURRENT",
           description: "By default we filter shows by current",
         },
+        includeStubShows: {
+          type: GraphQLBoolean,
+          description: "Whether to include local discovery stubs",
+        },
+        displayable: {
+          type: GraphQLBoolean,
+          defaultValue: true,
+          description: "Filter by displayable shows",
+        },
         discoverable: {
           type: GraphQLBoolean,
-          description:
-            "Whether to include local discovery stubs as well as displayable shows",
+          description: "Whether to include stub shows or not",
+          deprecationReason:
+            "Use `includeStubShows` and `displayable` combination",
```

Comment:
👍

Reply:
Oh wait I guess based on other comments, this line might be misleading. 

Should be sufficient to say "Use `includeStubShows`" I believe?